packages/core/src/constants.ts

```diff
@@ -8,9 +8,7 @@ export const DEFAULT_DOMAIN = 'cognitedata.com';
 export const BASE_URL = `https://${DEFAULT_CLUSTER}.${DEFAULT_DOMAIN}`;
 
 /** @hidden */
-export const AUTHORIZATION_HEADER = 'Authorization';
-/** @hidden */
-export const API_KEY_HEADER = 'api-key';
+export const AUTHORIZATION_HEADER = 'authorization';
```

Comment:
Feels like this change should break things but i'm not sure if we add both versions of the authorization header.

Reply:
Tests passed and the api shoudn’t care about the casing

Reply:
Or what are you worried about will break?

Reply:
> the api shoudn’t care about the casing

I'm worried about this part. If we're using the same constant everywhere in the repo (which we probably do), i'm not surprised tests are working fine. Do we have a case that we pass this header manually from the outside of the sdk?

Reply:
(this is not a blocker though, just nitpicking)

Reply:
Let me check how easy it is to undo the change. But we run integration tests so the API accepts the header

Reply:
Then don't worry too much about it, we would be testing this sdk version anyway (and it would be almost instant to figure it out)

Reply:
This is the only place in `fusion` repo that we manually pass the authorization header: https://github.com/cognitedata/fusion/blob/a3b007c8255cb0475878f6767e95a658ba8c34d9/libs/shared/cogidp-sdk/src/lib/client.ts#L226-L228. But we're not using js sdk there so should be good imo